### ISSUE TITLE
Implement Gluu Persistent Noncorreletable Identifier

### ISSUE THREAD

**Issue by nynymike** [GluuFederation/oxShibboleth#58]
Persistent non-correletable identifiers in SAML, or pairwise identifiers in OpeniD, are the same for the subject at a certain RP, but different for each RP.  In the past we used a Shib plugin that stored. These ids can be either algorithmically generated (APID) or stored on disk (PPID).  The latter is better if you need to search the database to figure out which person was issued a certain identifier. 

I think we should support PPIDs in Shibboleth, but store them the same way we store PPIDs in OpenID Connect.

**Comment by malotian** [comment]
@nynymike need to discuss on this,
persistent nameid - its same for subject (regardless of RP)
transient nameid - different for each SAML Transaction

issue mentioned is interesting - where we wanted something similar to persistent - but scoped to RP
it can be supported, but we would need to create custom generator (that we are capable of)

**Comment by nynymike** [comment]
Did you see the docs on this page: https://wiki.shibboleth.net/confluence/display/IDP30/PersistentNameIDGenerationConfiguration

**Comment by malotian** [comment]
@nynymike yes I have implemented same/similar in nameid, will discuss so that we are on same page as far as understanding is concerned, we need to experiment a bit and update documentation, from implementation perspective i guess we are good (will evaluate further)
